src/domain/organization/organizationForm/__tests__/OrganizationForm.test.tsx

```diff
@@ -28,19 +28,52 @@ const renderComponent = () =>
   render(<OrganizationForm />, { mocks: defaultMocks });
 
 const getElement = (
-  key: 'addMerchantButton' | 'deleteMercantButton' | 'paytrailMerchantIdInput'
+  key:
+    | 'addAccountButton'
+    | 'addMerchantButton'
+    | 'deleteAccountButton'
+    | 'deleteMercantButton'
+    | 'paytrailMerchantIdInput'
+    | 'vatCodeInput'
 ) => {
   switch (key) {
+    case 'addAccountButton':
+      return screen.getByRole('button', { name: 'Lisää uusi tili' });
     case 'addMerchantButton':
       return screen.getByRole('button', { name: 'Lisää uusi kauppias' });
+    case 'deleteAccountButton':
+      return screen.getByRole('button', { name: 'Poista tili' });
     case 'deleteMercantButton':
       return screen.getByRole('button', { name: 'Poista kauppias' });
     case 'paytrailMerchantIdInput':
       return screen.getByRole('textbox', { name: 'Paytrail-kauppiastunnus *' });
+    case 'vatCodeInput':
+      return screen.getByRole('textbox', { name: 'ALV-koodi *' });
   }
 };
 
-test('should add and remove price group', async () => {
+test('should add and remove account', async () => {
+  mockAuthenticatedLoginState();
```

Comment:
Optional suggestion (can merge without this in my opinion): SonarCloud seems to have picked up duplication, for example in these tests. Maybe the tests could benefit from scoping with `describe` or from a `beforeEach` to have the initialization steps repeated only once?